src/Table/Table.component.tsx

```diff
@@ -276,6 +282,23 @@ const SWrapperDiv = styled(ReactTable)`
   &.ReactTable .rt-noData {
     display: none !important;
   }
+
+  ${props =>
+    props.stickyHeaders &&
+    css`
+      &.ReactTable .rt-table {
+        overflow: visible;
+        border-top: none;
+      }
+
+      &.ReactTable .rt-thead.-header {
+        position: sticky;
+        top: 0;
+        z-index: 1;
+        background-color: white;
+        border-top: ${props => props.theme.table.border};
```

Comment:
Is the `border-top:` needed? Seems odd that we'd need to define it here

Reply:
Yes, it's needed to prevent an issue where the top border of the header wasn't sticking. I think the border was attached to the table and not the header row which was what was sticking to the top. So the top table border scrolled out of view which is why I set the border-top of the table to none. So basically we are just defining the top border in a place we can control more easily.